.github/workflows/macosx.yml

```diff
@@ -10,11 +10,12 @@ jobs:
   build:
     strategy:
       matrix:
+        cxxstd: [3, 11]
         include:
           - os: macos-10.15
             platform_file: include $(ACE_ROOT)/include/makeinclude/platform_macosx.GNU
     runs-on: ${{ matrix.os }}
-    name: ${{ matrix.os }}
+    name: "${{ matrix.os }}-${{ matrix.cxxstd }}"
```

Comment:
Maybe add C++ in the name before the number, that way someone can more easily determine what 3/11 mean, use at that moment 03 instead of 3